app/Cardano/Shell/Features/Logging.hs

```diff
@@ -78,6 +80,7 @@ data LoggingLayer = LoggingLayer
     , llBracketStmIO    :: forall a t.      (Show a) => Trace IO a -> Severity -> Text -> STM.STM t -> IO t
     , llBracketStmLogIO :: forall a t.      (Show a) => Trace IO a -> Severity -> Text -> STM.STM (t,[(LOMeta, LOContent a)]) -> IO t
     , llConfiguration   :: Configuration
+    , llAddBackend      :: Backend Text -> Text -> IO ()
```

Comment:
`Either ...` with the error being the error that can occur when a user selects the SwitchBoard multiple times or something like that?

Reply:
the user does not have access to the switchboard, so cannot connect it again.
but, yes this would be uncomfortable. we will see if other error cases can occur.